<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="4">

  <testsuite name="thrust-engine-demo-test"
    errors="0" failures="1" hostname="" id="" package="thrust-engine-demo-test" skipped="0"
    tests="4" time="0.000" timestamp="Thu, 24 Dec 2015 11:10:10 GMT">
  
    <testcase name="Phaser Game Boots Test" time="33.99" assertions="2">

    

    
    </testcase>
  
    <testcase name="Phaser Game Loads Test" time="3.343" assertions="4">

    

    

    

    
    </testcase>
  
    <testcase name="E2E Actors are avaialable to test" time="4.282" assertions="3">

    

    

    
    </testcase>
  
    <testcase name="Demo Player Control from Tests" time="20.04" assertions="1">  <failure message="Testing if Player is dead.">    at Object.module.exports.Demo Player Control from Tests (/Users/hayesmaker/Workspace/hayesmaker/thrust-engine/test/e2e/nightwatch/tests/thrust-engine-demo-test.js:87:15)
    at Module.call (/usr/local/lib/node_modules/nightwatch/lib/runner/module.js:63:34)
    at /usr/local/lib/node_modules/nightwatch/lib/runner/testcase.js:97:29
    at _fulfilled (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:796:13)
    at /usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:556:49
    at runSingle (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:137:13)
    at flush (/usr/local/lib/node_modules/nightwatch/node_modules/q/q.js:125:13)
    at nextTickCallbackWith0Args (node.js:433:9)</failure>

    
    </testcase>
  
  
  </testsuite>
</testsuites>
